integrations/datadog/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
+
+# 2024-02-06
+
+### Features
+
+- Implemented Datadog integration (1.0.0)
```

Comment:
Implemented Datadog integration to bring host, monitor, service catalog and SLO entities (PORT-5633)